### DISCUSSION TITLE
Q&A capability

### DISCUSSION THREAD

**Discussion by jalkestrup** [AI4Finance-Foundation/FinGPT#147] [Q&A]
Hi AI4Finance - first of all thanks for the awesome work being put into FinGPT and all the documentation shared.

As I understand, most of the recent models, including the https://huggingface.co/FinGPT/fingpt-mt_llama2-7b,_lora is finetuned on the entire Fin dataset, including this Q&A task, https://huggingface.co/datasets/FinGPT/fingpt-fiqa_qa ,  that has fairly long string outputs as the objective.

How come the current model only seem to support 'Financial Sentiment Analysis', 'Financial Relation Extraction', 'Financial Headline Classification' & 'Financial Named Entity Recognition'? - and provide very brief yes/no answers to any prompt?

For example, if I feed one of the training examples from the QA dataset, 

```python
prompt = [
    '''Instruction: UOffer your insights or judgment on the input financial query or topic using your financial expertise. Reply as normal question answering.
    Input: How can I register a UK business without providing a business address?
    Answer: '''
]
```

And the output is: "No"

Other similar prompts provide equally short yes/no outputs.

Thanks for helping clarify this,

**Comment by artist-genai** [comment]
Did you ever found an answer to this question?
